cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,71 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
+    cy.get('#alertButton')
+      .click();
 
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('You clicked a button');
+    })
   });
 
   it('should have the ability to assert scheduled allert', () => {
+    cy.get('#timerAlertButton')
+      .click();
 
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('This alert appeared after 5 seconds');
+    });
   });
 
   it('should autimatically resolve alerts', () => {
+    cy.get('#confirmButton')
+      .click();
+
+    cy.on('window:confirm', (str) => {
+      expect(str).to.equal('Do you confirm action?');
+      return true;
+    });
 
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('You selected Ok');
```

Comment:
you need to select it by cy.get('#confirmResult'). You method doesn't work for me